tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/op/core/ShapeOps.java

```diff
@@ -0,0 +1,217 @@
+/*
```

Comment:
Please add the right license header (i.e. the non-Oracle one that references Tensorflow authors).

Reply:
OK